styles.css

```diff
@@ -0,0 +1,642 @@
+:root {
+    --white: #ffffff;
+    --primary: #0b5351;
+    --primary-shadow: #719594;
+    --secondary: #9a8f97;
+    --secondary-shadow: #9a8f9799;
+    --tertiary: #092327;
+    --highlight: #c1d7ae;
+    /*formatting*/
+    --curve: 20px;
+}
+/*
+Inter,sans-serif
+*/
+/*ELEMENTS*/
+body {
+    background-color: white;
+    font-family: /*Inter,sans-serif*/'Poppins', sans-serif;
+    margin: 0;
+}
+
+h1{
+    font-family: 'Raleway', sans-serif;
+    font-weight: 700;
+    margin-bottom:5%;
+    margin-top:1%;
+}
+
+h2, h3, h4, h5{
+    font-family: 'Raleway', sans-serif;
+    font-weight: 600;
+    margin:2% 0%;
+}
+
+p, li {
+    text-wrap: balance;
+    font-family: Inter,sans-serif;
+}
+
+a {
+    /* padding-bottom: 3%; */
+    text-decoration-line: none;
+}
+
+section {
+    display: flex; /*Displays divs nicely in each section*/
+    flex-direction: column; /*Keeps section titles above boxes*/
+    align-items: center; /*centers divs on the vertical axis*/
+    height: 100vh;
+    /* max-height: 650px; */
+    /* padding-top: 30px; */
+}
+
+/*Section titles for each section*/
+.title {
+    margin: 2%;
+    padding-top: 30px;
+}
+
+
+/*Navigation bar*/
+
+nav {
+    display: flex; /*Keeps all items on singular line*/
+     /*Equally spaces out div elements in nave bar*/
+    justify-content: space-between;
+    position: fixed; /*Mean nav bar will always be visible even when scrolling*/
+    left: 0px;
+    right: 0px;
+    z-index: 10;
+    /* padding-bottom: 5px; */ /*Keeps nave bar looking nice*/
+    border-radius: 0px 0px var(--curve) var(--curve);
+    background-color: var(--tertiary);
+}
+nav #menu-icon{
+    display: none;
+}
+
+nav ul {
+    /*keeps nave bar dimensions to nav bar id*/
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+nav ul li{
+   /*Removes bullet points*/
+  float: left; /*Keeps list horizontal*/
+}
+
+nav ul a, nav ul li, #name, #nav-contact, nav div a{
```

Comment:
Ideally you shouldn't need to style both an a and a li within a nav just style one